bin/colrm

```diff
@@ -14,31 +14,57 @@ License: perl
 
 use strict;
 
-$0 =~ s(.*/)();
-my $usage = "usage: $0 [startcol endcol]\n";
-my ($startcol, $endcol);
+use File::Basename qw(basename);
 
-# I could be more clever, but this will run faster
+my $Program = basename($0);
 
 if (@ARGV > 2) {
-	die $usage;
+	die "usage: $Program [startcol [endcol]]\n";
```

Comment:
let's change these to a `warn` and exit with the right value for the real colrm in the same situation

Reply:
Ok. The netbsd and openbsd manuals didn't help much but the openbsd source does exit(1) within usage() function.

Reply:
I usually go with whatever I discover is in $? when I try the real program :)